### PR TITLE
Allow connectivity to GCP WC

### PR DESCRIPTION
**Pull request by fiunchinho**

### CODE REVIEW

File: tekton/tasks/create-cluster-capi-pure.yaml [hunk 1]
```diff
@@ -174,6 +174,12 @@ spec:
       echo -n "${organization}" > $(results.organization.path)
 
       kubectl apply -f /tmp/cluster.yaml
+      
+      # We need to allow the IP of the cluster where this tekton task is running.
+      # We should remove this IF and add flags to kubectl-gs instead.
+      if [ "$PROVIDER" = "gcp" ]; then
+        kubectl annotate -n org-$organization gcpcluster ${cluster_id} api.gcp.giantswarm.io/allowlist=194.107.128.0/24
```
Comment (calvix) [inline tekton/tasks/create-cluster-capi-pure.yaml:181]: `194.107.128.0/24` what is that supposed to be?

Comment (calvix) [reply tekton/tasks/create-cluster-capi-pure.yaml:181]: that super wide range, why not allow the 3 NAT gateway IPs?

Comment (fiunchinho) [reply tekton/tasks/create-cluster-capi-pure.yaml:181]: gorilla/rfjh2 IP's. Should I add it to the comment?

Comment (fiunchinho) [reply tekton/tasks/create-cluster-capi-pure.yaml:181]: I took it from an old PR that was allowing the same cluster.

Comment (fiunchinho) [reply tekton/tasks/create-cluster-capi-pure.yaml]: Wait, I was using a wrong IP. Please, take a look now.

Comment (calvix) [reply tekton/tasks/create-cluster-capi-pure.yaml]: yes this is more like it

**Review by calvix — APPROVED** [review]
this looks better :)

**Pull request merged**